allennlp/data/dataset_readers/dataset_utils/text2sql_utils.py

```diff
@@ -76,32 +85,42 @@ def process_sql_data_blob(data: JsonDict,
     use_all_sql : ``bool``, optional (default = False)
         Whether to use all of the sql queries which have identical semantics,
         or whether to just use the first one.
+    use_all_queries : ``bool``, (default = False)
```

Comment:
This doesn't match the parameter name; above it's `use_unique_queries`.  I'd vote for keeping `use_all_queries`, and do `if not use_all_queries` where you have `use_unique_queries` below.